src/overlays/actors/ovl_Door_Spiral/z_door_spiral.c

```diff
@@ -31,30 +40,221 @@ const ActorInit Door_Spiral_InitVars = {
     (ActorFunc)DoorSpiral_Init,
     (ActorFunc)DoorSpiral_Destroy,
     (ActorFunc)DoorSpiral_Update,
-    (ActorFunc)DoorSpiral_Draw
+    (ActorFunc)DoorSpiral_Draw,
+};
+
+typedef struct {
+    /* 0x0 */ Gfx* unk_00;
+    /* 0x4 */ Gfx* unk_04;
+    /* 0x8 */ u32 unk_08;
+    /* 0x9 */ u8 unk_09;
+    /* 0xA */ u8 translateZ;
+    /* 0xB */ u8 unk_0B;
+    /* 0xC */ u8 unk_0C;
+} SpiralInfo; // size = 0xD
```

Comment:
```suggestion
typedef struct {
    /* 0x0 */ Gfx* unk_00;
    /* 0x4 */ Gfx* unk_04;
    /* 0x8 */ u32 unk_08;
    /* 0x9 */ u8 unk_09;
    /* 0xA */ u8 translateZ;
    /* 0xB */ u8 unk_0B;
    /* 0xC */ u8 unk_0C;
} SpiralInfo; // size = 0x10
```
Because of alignment, I think this struct would actually be size 0x10